doc/user_guide/install.rst

```diff
@@ -178,7 +178,7 @@ dependencies required by specific functionalities:
   GUI elements,
 * ``gui-traitsui`` to use the GUI elements based on `traitsui <https://docs.enthought.com/traitsui/>`_,
 * ``mrcz`` to read mrcz file,
```

Comment:
Although it is unrelated to this PR, it seems that several of the extras listed here have now moved to rsciio.

Reply:
Fixed in b45ab6b